build_tools/cmake/iree_bitcode_library.cmake

```diff
@@ -227,6 +227,98 @@ function(iree_cuda_bitcode_library)
   )
 endfunction()
 
+# iree_amdgpu_bitcode_library()
+#
+# Builds an AMDGPU LLVM bitcode library from an input file via clang.
+#
+# Parameters:
+# NAME: Name of the target.
+# GPU_ARCH: Target AMDGPU architecture, e.g. gfx942.
+# SRCS: Source files to pass to clang. Headers (*.h) are for dependency
+#       tracking only. Current limitation: only one non-header source is
+#       supported.
+# COPTS: Additional flags to pass to clang.
+# OUT: Output file name. Defaults to {source.c}.{gpu_arch}.bc.
+#
+function(iree_amdgpu_bitcode_library)
```

Comment:
I tested a build of `all` on my Windows system to confirm that this works there. No issues building, and got these in the logs:
```
[build] [4656/5157  89% :: 791.454] Compiling iree_uk_amdgpu_argmax_f32i64.c to iree_uk_amdgpu_argmax_f32i64.c.gfx90a.bc
[build] [4657/5157  89% :: 791.463] Compiling iree_uk_amdgpu_argmax_f16i64.c to iree_uk_amdgpu_argmax_f16i64.c.gfx90a.bc
```

Reply:
thanks for checking!